lib/stat_tracker.rb

```diff
@@ -1,16 +1,20 @@
 require 'csv'
 class StatTracker
   attr_reader :game_teams,
-              :games,
+              :game_rows,
               :teams
 
   def initialize(locations)
-    game_teams = CSV.read locations[:game_teams], headers: true, header_converters: :symbol
-    games = CSV.read locations[:games], headers: true, header_converters: :symbol
-    teams = CSV.read locations[:teams], headers: true, header_converters: :symbol
```

Comment:
We do need to add a '@' in front of these variables to make them instance variables.